t/lib/MT/Test.pm

```diff
@@ -1998,6 +1998,11 @@ sub has_php {
     my $php_version_string = `php --version 2>&1` or return $HasPHP = 0;
     my ($php_version) = $php_version_string =~ /^PHP (\d+\.\d+)/i;
     $HasPHP = ( $php_version and $php_version >= 5 ) ? 1 : 0;
+    if (MTM->config->ObjectDriver eq 'DBD::MSSQLServer') {
```

Comment:
@charsbar s/MTM/MT/ ?

Reply:
oops. Thanks!

Reply:
@charsbar 
should check like this.
> MT->config->ObjectDriver =~ /u?mssqlserver/i

mt-config.cgi generated by mt-wizard.cgi uses config_package registry as ObjectDriver value.
https://github.com/movabletype/movabletype/blob/84aea1f4d5c364583c79c64ac15abae00a15e6fc/lib/MT/App/Wizard.pm#L1195

and MT::ObjectDriverFactory uses match registry to check ObjectDriver value.
https://github.com/movabletype/movabletype/blob/84aea1f4d5c364583c79c64ac15abae00a15e6fc/lib/MT/ObjectDriverFactory.pm#L86

Reply:
Not sure if we need to care about mt-wizard.cgi here as this is only for testing, but it certainly seems better to support UMSSQLServer driver as well. Thanks!